sycl/source/detail/pi.cpp

```diff
@@ -140,12 +140,10 @@ void assertion(bool Condition, const char *Message) {
 // implemented yet.
 
 #define _PI_API(api)                                                           \
-  template <>                                                                  \
-  CallPi<decltype(&::api),                                                      \
-        (offsetof(pi_plugin::FunctionPointers, api))>::CallPi() {               \
+  template <> CallPi<decltype(&::api)>::CallPi() {                             \
```

Comment:
where had the offset parameter gone?

Reply:
Sorry about that. I changed the code to check the exact error message I get; changed it back. tested it, but during cleaning of the code. Did this mistake.